modules.d/95rootfs-block/mount-root.sh

```diff
@@ -30,21 +30,12 @@ mount_root() {
         fsck_ask_err
     done
 
-    READONLY=
     fsckoptions=
     if [ -f "$NEWROOT"/etc/sysconfig/readonly-root ]; then
         # shellcheck disable=SC1090
         . "$NEWROOT"/etc/sysconfig/readonly-root
     fi
 
-    if getargbool 0 "readonlyroot=" -y readonlyroot; then
-        READONLY=yes
-    fi
-
-    if getarg noreadonlyroot; then
-        READONLY=no
-    fi
-
     if [ -f "$NEWROOT"/fastboot ] || getargbool 0 fastboot; then
```

Comment:
I'd also remove the `fastboot` part, it provides the same functionality as `rd.skipfsck`.